frontend/packages/console-shared/src/components/getting-started/__tests__/QuickStartGettingStartedCard.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { GettingStartedCard } from '@console/shared/src/components/getting-started';
+import { useActiveNamespace } from '@console/shared/src';
+
+import QuickStartsLoader from '@console/app/src/components/quick-starts/loader/QuickStartsLoader';
+
+import { QuickStartGettingStartedCard } from '../QuickStartGettingStartedCard';
+import { loadingQuickStarts, loadedQuickStarts } from './QuickStartGettingStartedCard.data';
+
+jest.mock('react', () => ({
+  ...require.requireActual('react'),
+  useLayoutEffect: require.requireActual('react').useEffect,
+}));
```

Comment:
Are you using `useLayoutEffect` somewhere and why is it assigned `useEffect`?

Reply:
You're right, it was not used here and so I removed this here.

I keep that in `GettingStartedGrid.spec.tsx` and `RestoreGettingStartedButton.spec.tsx` and added a comment there why this was required. It was required for some PatternFly components (Popover and Label) which use `useLayoutEffect` internally. It removes only a warning while running the tests.